### DISCUSSION TITLE
custom domain 403 forbidden

### DISCUSSION THREAD

**Discussion by sharaku17** [vercel/community#330] [Help] [answered]
Hi,

I purchased a domain and added the DNS  record so that my application deployed on vercel is using my custom domain. I added the A Record like described in the process. Now if I visit the domain I get greeted by a 403 error page that says "Forbidden" access to this resource on the server is denied.

The App is working fine under "name.vercel.app"  but I can not get it to work using my own custom domain somehow. What am I doing wrong?

**Comment by samcx** [comment]
@sharaku17 Can you share with us the name of this custom domain so we can take a look?

**Comment by Pieparker** [comment, marked as answer]
This conversation appears to have gone quiet. To keep discussions on topic and to prevent the resurrection of old topics, this discussion has been closed. If issues persist or a future reader stumbles upon this discussion and is exhibiting the same problem, please create a new discussion.
